Basic/Basic/OrganizationInfoViewController.swift

```diff
@@ -33,4 +37,47 @@ class OrganizationInfoViewController: UIViewController
         alert.addAction(okAction);
         self.present(alert, animated: true, completion: nil);
     }
+    
+    @IBAction func clickFetchOrganizations(_ sender: UIButton)
+    {
+        let appDel = AppDelegate.instance()
+        let context = appDel!.persistentContainer.viewContext
+        let coord = appDel?.persistentContainer.persistentStoreCoordinator
+        
+        let fetchRequest = NSFetchRequest<NSFetchRequestResult>(entityName: "OrganizationModel")
+        let deleteRequest = NSBatchDeleteRequest(fetchRequest: fetchRequest)
+        
+        do {
+            try coord?.execute(deleteRequest, with: context)
+        } catch let error as NSError {
+            debugPrint(error)
+        }
+        
+        RequestManager.fetchOrganizations() { response in
+            let organizations = RequestManager.parseToOrganizationArray(json: response!)
```

Comment:
For response you're using optional unwraps everywhere, but here you just do force unwrap. What it fails somewhere? Add optional unwrap here as well.